### ISSUE TITLE
[Feature] Add default image's height/width in Markdown

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav#3235]
Google has started to measure the [Cumulative Layout Shift](https://web.dev/cls/) in its Lighthouse dev tool.

Markdown and Twig already allow to add width/height parameters (albeit undocumented afaik).

In Twig the functions `height()` and `width()` without parameters use the dimensions of the provided image.
Twig:
```
{{ page.media.images['image.jpg'].width().height().html("My Title", "My Alt") | raw }}
```
Generates:
```
<img width="1200" height="800" title="My Title" alt="My Alt" src="/user/pages/01.home/image.jpg" />
```

However, Markdown does not provide the image's dimensions:
Markdown:
```
![My Alt](image.jpg "My Title")
![My Alt](image.jpg?width&height "My Title")
```
Generates:
```
<img title="My Title" alt="My Alt" src="/user/pages/01.home/image.jpg"
<img width="1" height="1" title="My Title" alt="My Alt" src="/user/pages/01.home/image.jpg" />
```

Feature request:
- When height/width are provided in Markdown without parameters, use dimensions of provided image. 
Why? Because editors are not likely (imho) to know the dimensions of the image they add to Markdown.
- Even better: Consider a way that if height/width are not provided in (Twig and) Markdown, the width/height are added anyway using the image's dimensions. Unless as config setting tells it not to.

**Comment by w00fz** [comment]
Agreed this is something that needs to be investigated and implemented as CLS has become a ranking factor.

**Comment by pamtbaau** [comment]
I've created an implementation proposal in PR #3272

**Comment by Rotzbua** [comment]
@w00fz @rhukster PR merged. Think this can be closed. Great feature 👍
